css/index.css

```
@@ -150,12 +162,108 @@ footer p {
 }
 
 
-@media all and (max-width: 1200px) {
+
+
+/* style navigational header */
+header ul {
+
+    margin: 100px;
+    font-family: Verdana;
+    font-weight: bold;
+    display: inline;
+    background-color: rgb(136, 6, 136);
+    border: 5px solid black;
+    box-shadow: 0 25px 25px -25px black;
+}
+
+/* made header text next to eachother with space with padding*/
+header ul li {
+
+    display: inline-block;
+    cursor: auto;
+    padding: 13px 35px;
+    list-style: none;
+    position: relative;
+    z-index: 5;
+    transition: 0.3s;
+}
+
+/*hover nav bar*/
+header ul li:hover {
+
+    background: plum;
+}
+
+/* made header with no underline & change link text color*/
+header li a {
+
+    text-transform: uppercase;
+    text-decoration: none;
+    color: black
+}
+
+
+/* made header sticky*/
+header {
+
+    overflow: hidden;
+    position: sticky;
+    top: 0;
+    left: 0;
+}
+
+/*make scrolling smooth*/
+html {
+    scroll-behavior: smooth;
+}
+
+/*added flex with columns*/
+section h2,
+h3,
+ul,
+li,
+form {
+
+
+    flex-direction: column;
+
+
+    display: flex;
+
+}
+
+/*media query for all desktops*/
+
+@media (max-width: 1200px) {
 
     section,
     body {
+        flex-wrap: wrap;
         background-color: teal;
         border: 20px solid grey;
 
     }
+}
+
+/* media query for mobile Phones */
+@media only screen and(max-width : 480px) {
```

there should be a space after `and` and there should *not* be a space **before** the colon `:` symbol

```suggestion
@media only screen and (max-width: 480px) {
```